.github/workflows/main.yml

```diff
@@ -38,6 +43,24 @@ jobs:
       - name: Build package and generate docs
         run: npm run generate
 
+      # Running tests
+      - name: Create .env files in submodule
+        run: node test/ci-setup-env.js
+        env: 
+          TILLED_SECRET_KEY: ${{ secrets.TILLED_SECRET_KEY }}
+          TILLED_PARTNER_ACCOUNT: ${{ secrets.TILLED_PARTNER_ACCOUNT }}
+          VITE_TILLED_PUBLIC_KEY: ${{ secrets.TILLED_PUBLIC_KEY }}
+          VITE_TILLED_MERCHANT_ACCOUNT_ID: ${{ secrets.TILLED_MERCHANT_ACCOUNT_ID }}
+          VITE_TILLED_CUSTOMER_ID: ${{ secrets.TILLED_CUSTOMER_ID }}
+          CI: true
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+      - name: Run Playwright tests
+        run: npx playwright test
+      - name: Print Success
+        run: echo "Success!"
```

Comment:
this isn't needed

Reply:
Sorry, I added this during testing, I'll take it out 😅